src/alarms/alarmgroups/NewGroup.tsx

```diff
@@ -1,7 +1,16 @@
 import React from "react";
 
 import GroupForm from "./GroupForm";
+import SpinnerIfNotLoaded from '../../components/SpinnerIfNotLoaded';
 
-export const NewGroup: React.FC = () => {
-  return <GroupForm />;
+
+export const NewGroup = () => {
+    return (
+      <SpinnerIfNotLoaded
+        loaded={true}
```

Comment:
can just say `loaded`

Reply:
Also, in this case, it is not necessary to wrap the Form component inside the `SpinnerIfNotLoaded` anymore but you can directly return the Form as the form is not waiting for anything to be loaded.